simphony/io/cuds_file.py

```diff
@@ -85,6 +85,58 @@ def close(self):
         """
         self._file.close()
 
+    def add_lattice(self, name, lattice, keywords=[]):
+        """Add lattice to the file.
+
+        Parameters
+        ----------
+        name : str
+            name of lattice
+        lattice : Lattice
+            lattice to be added.
+        keywords : list of str, optional
```

Comment:
I do not see the `keywords` argument used in the code, it does not look that it is necessary.